core-services/prow/02_config/redhat-appstudio/image-controler/_prowconfig.yaml

```diff
@@ -0,0 +1,6 @@
+branch-protection:
```

Comment:
hmm I think this is actually a wrong file

see https://github.com/openshift/release/blob/57dc33ea6e250f09f38fead1398fed97944c482c/core-services/prow/02_config/redhat-appstudio/image-controller/_prowconfig.yaml